opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -20,113 +20,154 @@
  */
 public interface Tracer {
```

Comment:
The indenting in this file was off, so I fixed it. Maybe I'll submit and merge that as a separate PR, as the diff here isn't nearly as large as it appears.